### ISSUE TITLE
[BUG] Lose language highlighting in other view - go lang

### ISSUE THREAD

**Issue by hughfe** [notepad-plus-plus/notepad-plus-plus#15482]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

Have two go lang files open, one moved to other view, after a little editing, all text in one of the documents is formatted as comment (gray).

### Steps To Reproduce

1.  Language selection go
2.  Open 2 documents, move one to "other view"
3.  Edit one, in particular delete a line or two
4. Chunks of one document are gray - formatted as comments.
![temp](https://github.com/user-attachments/assets/15a06b8b-8281-407e-a376-7e522d1d854e)

### Current Behavior

Program content formatted as comment

### Expected Behavior

Program content should be formatted as normal

### Debug Information

```shell
Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : OFF
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.4651
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.2)
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

**Comment by notdodgeball** [comment]
I can't make it happen with version v8.7.1.

Do you have sample files where this happens? And even better, the steps to reproduce it.

**Comment by hughfe** [comment]
Thanks for looking into this.  I was using an earlier version of Notepad++ that did not have support for Go and I found "go_byAnthonyStarks.xml", which did the job but it had this odd problem.  This even carried on with 8.7.1 but it was still using the 3rd party Go support.  At the same time I did a new install of Notepad++ on a different PC and found that it included support for Go (which worked properly).  I had to delete some config files and reinstall Notepad++ to get it to use the built-in Go support, which now seems to be ok (just deleting the "go_byAnthonyStarks.xml" left me with no Go formatting at all).
